src/main/java/games/strategy/engine/lobby/server/ModeratorController.java

```diff
@@ -2,12 +2,12 @@
 
 import java.util.Date;
 
-import games.strategy.engine.lobby.server.userDB.BannedMacController;
-import games.strategy.engine.lobby.server.userDB.BannedUsernameController;
+import games.strategy.engine.lobby.server.userdb.BannedMacController;
+import games.strategy.engine.lobby.server.userdb.BannedUsernameController;
 import games.strategy.engine.lobby.server.userDB.DBUser;
```

Comment:
Checkstyle: "Wrong lexicographical order for 'games.strategy.engine.lobby.server.userDB.DBUser' import. Should be before 'games.strategy.engine.lobby.server.userdb.BannedUsernameController'."